tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -134,6 +134,20 @@ module.exports = {
       return this.waitForElementVisible('@batchDeclineSharesButton')
         .click('@batchDeclineSharesButton')
         .waitForAjaxCallsToStartAndFinish()
+    },
+    isBatchButtonExist: async function (batchButtonName) {
+      let result = false
+      const batchButtonEl = {
+        selector: this.getbatchButtonEl(batchButtonName),
+        locateStrategy: 'xpath'
+      }
+      await this.isVisible(batchButtonEl, (res) => {
+        result = res.value
+      })
+      return result
```

Comment:
```suggestion
      return !!(result)
```
just return true or false for visibility

Reply:
**result** return **true** or **false** value. Do i need to do suggested change?

Reply:
if the return value is boolean for the visible and invisible state, your code works fine. no need to add suggested change.